core/src/banking_stage.rs

```diff
@@ -51,7 +51,7 @@ type PacketsAndOffsets = (Packets, Vec<usize>);
 pub type UnprocessedPackets = Vec<PacketsAndOffsets>;
 
 /// Transaction forwarding
-pub const FORWARD_TRANSACTIONS_TO_LEADER_AT_SLOT_OFFSET: u64 = 4;
+pub const FORWARD_TRANSACTIONS_TO_LEADER_AT_SLOT_OFFSET: u64 = 1;
```

Comment:
Did this improve forwarding accuracy/how are we measuring that accuracy? Wonder why it was 4 before 🤔

Reply:
yea, it reduced confirmation times by 200ms.  i think it might need to be dynamic under load.  But that's not clear to me.